dotnet/README.md

```diff
@@ -0,0 +1,31 @@
+# Containerize a dotnet project
+
+This is the incomplete beginning of a dotnet image created by following [Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-debian10). Projects that use dotnet and ASP.net should use this image as their base.
```

Comment:
Please wrap lines in READMEs to 120 characters - they're read via `less` from the terminal more often than they're read in github.

Reply:
I've wrapped them to around 100 chars